### DISCUSSION TITLE
Vite使用WebWorker | 愧怍

### DISCUSSION THREAD

**Discussion by giscus** [kuizuo/blog#327] [General]
# Vite使用WebWorker | 愧怍

愧怍的个人博客

https://kuizuo.cn/blog/vite-webworker/

**Comment by doudouHubs** [comment]
这种方法在移动端web无法生效，目前没找到原因

**Reply by doudouHubs** [reply]
已经查清楚了，是因为worker进程的日志不会输出到vConsole，加载没问题
